data/json/items/book/chemistry.json

```diff
@@ -0,0 +1,273 @@
+      "str": "PE023 \"Medical\": Application and Findings",
+      "str_pl": "copies of PE023 \"Medical\": Application and Findings"
+    },
+    "description": "This binder of highly technical papers describes some new chemical formula, and its effects on human subjects.  It's stamped \"APPROVED\"…",
+    "weight": "1934 g",
+    "volume": "1750 ml",
+    "price": 62000,
+    "price_postapoc": 1250000,
+    "bashing": 5,
+    "material": [ "paper", "plastic" ],
+    "symbol": "?",
+    "color": "light_green",
+    "skill": "chemistry",
+    "required_level": 8,
+    "max_level": 9,
+    "intelligence": 12,
+    "time": "35 m",
+    "fun": -2
+  },
+  {
+    "id": "recipe_raptor",
+    "type": "BOOK",
+    "name": { "str": "PE070 \"Raptor\": Proposal", "str_pl": "copies of PE070 \"Raptor\": Proposal" },
+    "description": "This sheaf of papers is a highly speculative proposal for focusing \"PE065\".  Scribbled notes throughout seem to think that it might work, but that there's no time.",
+    "weight": "50 g",
+    "volume": "500 ml",
+    "price": 125000,
+    "price_postapoc": 1250000,
+    "material": [ "paper" ],
+    "symbol": "?",
+    "color": "light_green",
+    "skill": "chemistry",
+    "required_level": 10,
+    "max_level": 10,
+    "intelligence": 12,
+    "time": "45 m",
+    "fun": -2
+  },
+  {
+    "id": "recipe_serum",
+    "type": "BOOK",
+    "name": { "str": "Best Practices for Compound Delivery", "str_pl": "copies of Best Practices for Compound Delivery" },
+    "description": "This internal manual details several varieties of mutagenic experiments, as well as describing the protocols used to concentrate mutagens for quicker results.  The authors recommend that researchers ensure that their subjects are well-fed and in good health, as the concentrated serums draw heavily on subjects' bodies.",
+    "weight": "1700 g",
+    "volume": "500 ml",
+    "price": 50000,
+    "price_postapoc": 1250000,
+    "material": [ "paper" ],
+    "symbol": "?",
+    "color": "light_green",
+    "skill": "chemistry",
+    "required_level": 8,
+    "max_level": 10,
+    "intelligence": 12,
+    "time": "45 m",
+    "fun": -4
+  },
+  {
+    "id": "reference_cooking",
+    "type": "BOOK",
+    "name": { "str": "CRC-Merck Handbook, 4th edition", "str_pl": "copies of CRC-Merck Handbook, 4th edition" },
+    "description": "This huge hardbound book is a collection of reference data and formulae pertinent to many technical disciplines.  If poring over tables of chemical and physical data is your thing, this is the book for you.",
+    "weight": "5790 g",
+    "volume": "1750 ml",
+    "price": 9000,
+    "price_postapoc": 30000,
+    "bashing": 7,
+    "material": [ "paper" ],
+    "symbol": "?",
+    "color": "light_green",
+    "intelligence": 4,
+    "time": "30 m",
+    "fun": -2
+  },
+  {
+    "id": "salt_chemistry",
+    "type": "BOOK",
+    "name": { "str": "Chemistry of Metallic Salts", "str_pl": "copies of Chemistry of Metallic Salts" },
```

Comment:
This isn't what we'd call this field, really, unless it was really novel (Molten salt chemistry, for example).

Perhaps describe it as an inorganic chemistry textbook, with an included lab manual section?

Reply:
Done. Probably gonna have to look into adding the book to more things, then.